crates/solver-types/src/standards/eip7683.rs

```diff
@@ -505,3 +505,37 @@ mod tests {
 		assert_eq!(deserialized.context.len(), 500);
 	}
 }
+
+// Solidity struct definitions for ABI encoding with OIF contracts
+#[cfg(feature = "abi")]
```

Comment:
Nit;This is a great idea, but maybe we should scope it to standard instead of ABI?